src/aiAgentsConfiguration/aiAgentsConfigurationTreeDataProvider.ts

```diff
@@ -17,13 +17,13 @@ export class AIAgentsConfigurationItem extends VSCode.TreeItem {
     public readonly id: string,
     public readonly label: string,
     public readonly isConfigured: boolean,
-    public readonly description?: string,
+    public readonly tooltipText?: string,
     public readonly configureCommand?: string,
     public readonly reconfigureCommand?: string
   ) {
     super(label, VSCode.TreeItemCollapsibleState.None);
     this.id = id;
-    this.description = this.getStatusDescription();
```

Comment:
I removed the description because the view is too narrow to display lots of horizontal text. Replaced it with a tooltip.